### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by qsomazzi**

### CODE REVIEW

**Comment by qsomazzi** [comment]
@rande ping

File: Tests/DependencyInjection/ConfigurationTest.php [hunk 1]
```diff
@@ -32,8 +32,9 @@ public function testClasses()
         $expected = array(
             'class' => array(
                 'dashboard' => 'MyApp\\Sonata\\DashboardBundle\\Entity\\Dashboard',
-                'block'     => 'Sonata\\DashboardBundle\\Entity\\Block',
+                'block'     => 'Application\\Sonata\\DashboardBundle\\Entity\\Block',
```
Comment (Soullivaneuh) [inline Tests/DependencyInjection/ConfigurationTest.php]: Why `Application\\`?

Comment (qsomazzi) [inline Tests/DependencyInjection/ConfigurationTest.php]: this is the default value, use for EasyExtends

Comment (qsomazzi) [inline Tests/DependencyInjection/ConfigurationTest.php]: like here : https://github.com/sonata-project/SonataPageBundle/blob/master/DependencyInjection/Configuration.php @Soullivaneuh

File: Tests/DependencyInjection/ConfigurationTest.php [hunk 2]
```diff
@@ -32,8 +32,9 @@ public function testClasses()
         $expected = array(
             'class' => array(
                 'dashboard' => 'MyApp\\Sonata\\DashboardBundle\\Entity\\Dashboard',
```
Comment (rande) [inline Tests/DependencyInjection/ConfigurationTest.php]: MyApp should also be Application

Comment (qsomazzi) [inline Tests/DependencyInjection/ConfigurationTest.php]: are you sure ? I override this value for the test here

Comment (rande) [inline Tests/DependencyInjection/ConfigurationTest.php]: sorry, ok for me

**Pull request merged**